### ISSUE TITLE
Accessibility on Popover

### ISSUE THREAD

**Issue by thomaswang** [dormdev/dormdev#8]
Dropdown `react-tiny-popover` should be WAI-ARIA compliant. 

Namely, should have better keyboard navigation:
- `aria-has-popup` set on the triggering element
- `esc` to close
- [arrow key navigation](http://simplyaccessible.com/article/arrow-key-navigation)
